NAMESPACE

```diff
@@ -1 +1,2 @@
 # Generated by roxygen2: do not edit by hand
+
```

Comment:
It looks like the exports are missing. Maybe run `devtools::document()` and then commit the results?